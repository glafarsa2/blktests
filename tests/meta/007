#!/bin/bash
# SPDX-License-Identifier: GPL-3.0+
# Copyright (C) 2017 Omar Sandoval
#
# Test requires().

. tests/meta/rc

DESCRIPTION="skip in requires()"

requires() {
	SKIP_REASON="(╯°□°)╯︵ ┻━┻"
	return 1
}

test() {
	echo '¯\_(ツ)_/¯'
}
