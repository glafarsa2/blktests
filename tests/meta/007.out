Running meta/007
Test complete
